frontend/src/views/reports/ReportingView.vue

```diff
@@ -265,6 +335,28 @@ export default {
       this.menuEndDate = false
       this.selectedFacility = this.currentFacility.facilityId
     },
+
+    async createSurveyResponse() {
+      try {
+        this.processing = true
+        const payload = {
+          contactId: this.userInfo?.contactId,
+          facilityId: this.selectedFacility,
+          surveyId: '16fb81de-6dc1-ee11-9079-000d3af4865d',
```

Comment:
I remember we discussed that we want a more readable survey Guid for monthly reporting, but I haven't seen any update on it. For now, I hardcoded survey Guid because it is the only report we have in CRM.

Reply:
Right. I forgot about that. I added a note and we can bring it up with CRM team. Fine for now.